mathesar/api/db/viewsets/records.py

```diff
@@ -49,7 +49,6 @@ def list(self, request, table_pk=None):
         column_names_to_ids = table.get_column_name_id_bidirectional_map()
         column_ids_to_names = column_names_to_ids.inverse
         if filter_unprocessed:
-            table = get_table_or_404(table_pk)
```

Comment:
What's going on here? Was this cruft left over from something?

Reply:
Yes